.github/workflows/build.yml

```diff
@@ -160,16 +160,19 @@ jobs:
             - uses: actions/setup-python@v2
               with:
                 python-version: '3.7'
+            - uses: lukka/get-cmake@latest
+              with:
+                cmakeVersion: 3.17.2
```

Comment:
This is what VVL does. We just leverage one of our existing CI builds to test the minimum.